src/types/transcript.ts

```diff
@@ -54,4 +55,10 @@ export interface TranscriptManager {
      * @param course course to request grade for
      */
     getGrade(studentID: StudentID, course: Course): number|undefined;
+
+    getCourseDescriptions(): CourseDescription[];
```

Comment:
Writing appropriate comments on what a function does would be very useful in future if you want to revisit the code.